examples/lighting-app/silabs/src/ZclCallbacks.cpp

```diff
@@ -51,16 +51,18 @@ void MatterPostAttributeChangeCallback(const chip::app::ConcreteAttributePath &
 #ifdef DIC_ENABLE
         dic_sendmsg("light/state", (const char *) (value ? (*value ? "on" : "off") : "invalid"));
 #endif // DIC_ENABLE
-        LightMgr().InitiateAction(AppEvent::kEventType_Light, *value ? LightingManager::ON_ACTION : LightingManager::OFF_ACTION,
-                                  value);
+        LightMgr().InitiateAction(AppEvent::kEventType_Light, *value ? LightingManager::ON_ACTION : LightingManager::OFF_ACTION);
     }
     // WIP Apply attribute change to Light
     else if (clusterId == LevelControl::Id)
     {
         ChipLogProgress(Zcl, "Level Control attribute ID: " ChipLogFormatMEI " Type: %u Value: %u, length %u",
                         ChipLogValueMEI(attributeId), type, *value, size);
 
-        LightMgr().InitiateAction(AppEvent::kEventType_Light, LightingManager::LEVEL_ACTION, value);
+        if (value != nullptr)
```

Comment:
You are calling InitiateLevelAction for any attribute change of level control cluster. 
Level action should only be done for CurrentLevel attribute changes no?

Reply:
Modified the condition to check for CurrentLevel attribute